includes/classes/class-omise-card-image.php

```diff
@@ -12,7 +12,7 @@ class Omise_Card_Image {
 		 */
 		private static function get_image( $file, $alternate_text ) {
 			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
-			return "<img src='$url/$file' width='38px' alt='$alternate_text' />";
+			return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

Comment:
Yes, but that solution can not solve the problem for a present customer on the related ticket.

And how about the non-technical customer that they can not, do not know or do not want to modify the style sheet?

We provide the best image size and those images are SVG file the responsive has enabled normally.